src/commons/error.rs

```diff
@@ -787,10 +812,28 @@ impl Error {
                 .with_ca(ca)
                 .with_auth(auth),
 
-            Error::RoaDeltaError(roa_delta_error) => {
-                ErrorResponse::new("ca-roa-delta-error", "Delta rejected, see included json")
-                    .with_roa_delta_error(roa_delta_error)
-            }
+            Error::RoaDeltaError(ca, roa_delta_error) => ErrorResponse::new("ca-roa-delta-error", &self)
+                .with_ca(ca)
+                .with_roa_delta_error(roa_delta_error),
+
+            //-----------------------------------------------------------------
+            // Autonomous System Provider Authorization - ASPA
+            //-----------------------------------------------------------------
```

Comment:
Above there are some "ca-aspas-" style names used, here it is "ca-aspa-", should these be consistent?

Reply:
It's consistent with how things were named for roas/roa. If we make this all consistent across the code base, then we will need to do data migrations (update keys) and it may result in breaking changes in the API with regards to error labels - although I don't think people actually use this.